distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/platform/TestConfiguration.java

```diff
@@ -624,9 +668,50 @@ public void testConfigStatusFailedImportReimportSuccessful() throws Exception {
                 is(true));
     }
 
+    private void enableCrls() throws IOException { 
+        FileUtils.moveFile(SERVER_ENCRYPTION_PROPERTIES.toFile(),
```

Comment:
Do you think we could have utility methods similar to `restoreBackup` that would hide all of that ugly code and get rid of all the duplication?